sdk/monitor/Azure.Monitor.Query/src/Models/LogsBatchQueryResultCollection.cs

```diff
@@ -33,32 +36,39 @@ public partial class LogsBatchQueryResults
         ///     &quot;AzureActivity | summarize Count = count() by ResourceGroup | top 10 by Count&quot;,
         ///     new DateTimeRange(TimeSpan.FromDays(1)));
         ///
-        /// Response&lt;LogsBatchQueryResults&gt; response = await client.QueryBatchAsync(batch);
+        /// Response&lt;LogsBatchQueryResultCollection&gt; response = await client.QueryBatchAsync(batch);
         ///
         /// var count = response.Value.GetResult&lt;int&gt;(countQueryId).Single();
         /// var topEntries = response.Value.GetResult&lt;MyLogEntryModel&gt;(topQueryId);
         /// </code>
         /// <param name="queryId">The query identifier returned from the <see cref="LogsBatchQuery.AddQuery"/>.</param>
-        /// <returns>The <see cref="LogsBatchQueryResults"/> with the query results.</returns>
+        /// <returns>The <see cref="LogsBatchQueryResultCollection"/> with the query results.</returns>
         /// <exception cref="ArgumentException">When the query with <paramref name="queryId"/> was not part of the batch.</exception>
         /// <exception cref="RequestFailedException">When the query <paramref name="queryId"/> failed.</exception>
         public LogsBatchQueryResult GetResult(string queryId)
         {
-            BatchQueryResponse result = Responses.SingleOrDefault(r => r.Id == queryId);
+            LogsBatchQueryResult result = this.SingleOrDefault(r => r.Id == queryId);
```

Comment:
Not sure if the description or the logic is more correct, but going by the description, I'd expect this to be an `Any` if the description is accurate

Reply:
Talked offline.